gen-art review part 1 (#314)
vcstuff/draft-ietf-oauth-status-list: draft-ietf-oauth-status-list.md

Status Lists can be used to express a variety of Status Types. This document defines basic Status Types for the most common use cases as well as an extensibility mechanism for custom Status Types.

> Furthermore, the document creates an extension point and IANA registry that enables other specifications to describe additional status mechanisms.

Reviewer: ```suggestion
Furthermore, the document creates an extension point and an IANA registry that enables other specifications to describe additional status mechanisms.
```

> Furthermore, the document creates an extension point and IANA registry that enables other specifications to describe additional status mechanisms.

Author: ```suggestion
Furthermore, the document creates an extension point and an IANA registry that enables other specifications to describe additional status mechanisms.
```

## Prior Work

Representing statuses with bits in an array is a rather old and well-known concept in computer science and there has been prior work to use this for revocation and status management such as a paper by Smith et al. {{smith2020let}} that proposed a mechanism called Certificate Revocation Vectors based on xz compressed bit vectors for each expiration day and the W3C bit Status List {{W3C.SL}} that similarly uses a compressed bit representation.

## Status Mechanisms Registry

> This specification establishes IANA "Status Mechanisms" registries for status mechanisms for JSON based tokens and for status mechanisms for CBOR based tokens and registers the members defined by this specification. Other specifications can register other members used for status retrieval.

Author: ```suggestion
This specification establishes IANA "Status Mechanisms" registries for status mechanisms for JOSE-based tokens and for status mechanisms for COSE-based tokens and registers the members defined by this specification. Other specifications can register other members used for status retrieval.
```
